refinery/data_set_manager/views.py

```diff
@@ -992,21 +992,24 @@ def put(self, request, uuid, format=None):
             return Response(message, status=status.HTTP_401_UNAUTHORIZED)
 
 
-class AddFilesToDataSetView(View):
+class AddFilesToDataSetView(APIView):
     """Add file(s) to an existing data set from upload directory or bucket"""
+    http_method_names = ['post']
+
     def post(self, request):
         try:
-            data_set = DataSet.objects.get(uuid=request.POST['data_set_uuid'])
+            data_set = DataSet.objects.get(uuid=request
+                                           .data.get('data_set_uuid'))
         except KeyError:
```

Comment:
I don't think this KeyError has the potential to be raised anymore

Reply:
@scottx611x Yes, that got updated when I wrote some unit tests.